state-chain/pallets/cf-threshold-signature/src/lib.rs

```diff
@@ -438,8 +438,13 @@ pub mod pallet {
 	/// State of the threshold signature requested.
 	#[pallet::storage]
 	#[pallet::getter(fn signature)]
-	pub type Signature<T: Config<I>, I: 'static = ()> =
-		StorageMap<_, Twox64Concat, RequestId, AsyncResult<SignatureResultFor<T, I>>, ValueQuery>;
+	pub type Signature<T: Config<I>, I: 'static = ()> = StorageMap<
+		_,
+		Twox64Concat,
+		RequestId,
+		(AggKeyFor<T, I>, AsyncResult<SignatureResultFor<T, I>>),
```

Comment:
I think the improvement in readability is pretty marginal since ```.0``` and ```.1``` was used in only in two places. I did the change anyways which was a bit of a pain to do. I couldnt get it to compile after changing the tuple to a struct in the storage item. couldnt figure out why but it was related to the ValueQuery. I therefore changed the ValueQuery to OptionQuery which meant some knock-on changes after which it compiled.